sdk/compute/Azure.ResourceManager.Compute/src/Customize/NetworkApiVersion.cs

```diff
@@ -7,14 +7,15 @@ namespace Azure.ResourceManager.Compute.Models
 {
     public readonly partial struct NetworkApiVersion
     {
-        /*
-        /// <summary> Network API version 2021-11-01. </summary>
         [CodeGenMember("TwoThousandTwenty1101")]
-        public static NetworkApiVersion v2021_11_01 { get; } = new NetworkApiVersion(TwoThousandTwenty1101Value);
-
-        /// <summary> Network API version 2022-11-01. </summary>
+        public static NetworkApiVersion v2020_11_01 { get; } = new NetworkApiVersion(TwoThousandTwenty1101Value);
         [CodeGenMember("TwoThousandTwentyTwo1101")]
         public static NetworkApiVersion v2022_11_01 { get; } = new NetworkApiVersion(TwoThousandTwentyTwo1101Value);
-        */
+
+        [CodeGenMember("TwoThousandTwenty1101")]
+        public static NetworkApiVersion TwoThousandTwenty1101 { get; } = v2020_11_01;
```

Comment:
unfortunately we have this member in our last version therefore we have to include this.
We should add a `[EditorBrowsable(Never)]` attribute on this.